tasks/omnibus.py

```diff
@@ -379,3 +386,26 @@ def manifest(
         omnibus_s3_cache=False,
         log_level=log_level,
     )
+
+@task
+def rpath_edit(ctx, install_path, target_rpath_dd_folder, macos=False):
+    for file in glob.iglob(f"{install_path}/**/*", recursive=True):
+        ext = os.path.splitext(file)[1]
+        if (ext != "" and "so" not in ext and "dylib" not in ext) or "json" in ext or not os.path.isfile(file):
```

Comment:
I think this can be split in multiple ifs to improve readability.
Instead of doing `"so" not in ext and "dylib" not in ext` you could probably have a list of wanted extensions and do `ext not in wanted_ext` instead, which would be clearer IMO

Reply:
Done in my upcoming commit 👍 , did a few changes on how the rpath_edit function worked since it was too slow on macOS.